makefile

```diff
@@ -65,28 +65,27 @@ unit-test: ## run unit test
 
 ########################################################
 build-local:  ## build dbpack cli, and put in dist dir
-	@mkdir -p dist
-	${GO_BUILD_ENVVARS} go build -o ./dist/dbpack ./cmd
+	${GO_BUILD_ENVVARS} go build -o dbpack ./cmd
 
 ########################################################
-build:  ## build dbpack cli, and put in dist dir
-	@mkdir -p dist
-	GOOS="linux"  GOARCH="amd64" CGO_ENABLED=0 go build -o ./dist/dbpack ./cmd
+docker-build: ## build docker image
+	go mod download
+	go mod tidy
+	docker build -f docker/Dockerfile-rws -t dbpack:latest .
```

Comment:
docker-build should use a common dockerfile

Reply:
how about importing docker build-tags?